### PR TITLE
[T] (quests) cyclic

### PR DESCRIPTION
**Pull request by luo-2233**
Co-authored-by: 疯子 <1161326997@qq.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/cyclic.snbt [hunk 1]
```diff
@@ -18,7 +18,7 @@
 			x: -7.5d
 			y: 0.5d
 			shape: "diamond"
-			description: ["A *very* powerful piece of technology that can right click with any item! [Try with a exnihilo sieve ;)]"]
+			description: ["一项*非常*强大的技术，可以右键单击任何物品！ [尝试对着筛子使用(*^_^*)]"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/cyclic.snbt]:
```suggestion
			description: ["一项*非常*强大的技术，可以右键单击任何物品！(尝试对着筛子使用)"]
```

File: config/ftbquests/quests/chapters/cyclic.snbt [hunk 2]
```diff
@@ -856,7 +856,7 @@
 			}
 			x: 1.0d
 			y: -1.0d
-			description: ["Cyclic is a great mod with lots of random bips and bops!"]
+			description: ["循环是一个很棒的mod，有多种多样的东西Cyclic is a great mod with lots of random bips and bops!"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/cyclic.snbt]:
```suggestion
			description: ["循环是一个很棒的模组，有林林总总的东西！"]
```

**Review by AnzhiZhang — APPROVED** [review]

File: config/ftbquests/quests/chapters/cyclic.snbt [hunk 3]
```diff
@@ -758,7 +758,7 @@
 		{
 			x: 11.5d
 			y: 0.5d
-			description: ["Prevents nearby sounds... good bye horrible loud noises!"]
+			description: ["隔绝附近的声音......再见了可怕的巨响！"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/cyclic.snbt]:
```suggestion
			description: ["隔绝附近的声音……再见了可怕的巨响！"]
```

**Review by Cactusstudent — APPROVED** [review]
没啥毛病

**Pull request merged**